tests/sentry/api/endpoints/test_organization_transaction_anomaly_detection.py

```diff
@@ -11,9 +11,9 @@
 from sentry.testutils.silo import region_silo_test
 
 
+@region_silo_test(stable=True)
 @freeze_time("2022-02-21")
-@region_silo_test
```

Comment:
In case it's not clear, `@region_silo_test` and `@freeze_time` needed to be in the opposite order to work.